pkg/helm/diff.go

```diff
@@ -54,17 +55,33 @@ func (d DiffObjectList) String() string {
 }
 
 // diffAgainstRelease returns the diff between the templated manifest and the manifest of the deployed Helm release.
-func diffAgainstRelease(restClientGetter genericclioptions.RESTClientGetter, namespace, manifest string, helmRelease *release.Release) (DiffObjectList, error) {
+func diffAgainstRelease(restClientGetter genericclioptions.RESTClientGetter, namespace string, helmTemplateRelease, helmRelease *release.Release) (DiffObjectList, error) {
 	remoteObjs, err := ObjectMapFromRelease(restClientGetter, helmRelease, nil)
 	if err != nil {
 		return nil, err
 	}
 
-	localObjs, err := ObjectMapFromManifest(restClientGetter, namespace, manifest, nil)
+	for _, hook := range helmRelease.Hooks {
+		hookObjs, err := ObjectMapFromManifest(restClientGetter, helmRelease.Namespace, hook.Manifest, nil)
+		if err != nil {
+			return nil, err
+		}
+		maps.Copy(remoteObjs, hookObjs)
```

Comment:
This copies from `remoteObjects` to `hookObjs`. We can expect that `remoteObjs` will be larger in most cases, so  reversing the parameters to `map.Copy()` should result in less operations.

Reply:
`maps.Copy` function is defined as:
```go
// Copy copies all key/value pairs in src adding them to dst.
// When a key in src is already present in dst,
// the value in dst will be overwritten by the value associated
// with the key in src.
func Copy[M1 ~map[K]V, M2 ~map[K]V, K comparable, V any](dst M1, src M2) {
	for k, v := range src {
		dst[k] = v
	}
}
```
This copies from `hookObjs` to `remoteObjs` so we already have less operations.